imobil.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Imobil extends Model
+{      
+    use HasFactory;
+    protected $fillable = [
+        'owner_id',
+        'type',
+        'price',
+        'rooms_nr',
+        'building_type',
+        'square_mp',
+        'floor',
+        'bath',
+        'balcony',
+        'description',
+        'created_at',
+    ];
+
+
+    public function address()
+    {
+        return $this->belongsTo(Address::class);
```

Comment:
here you should use `hasOne` and in address `belongsTo`

Reply:
public function address()
    {
        return $this->hasOne(Address::class);
    }